config/core/configmaps/features.yaml

```
@@ -47,8 +47,9 @@ data:
     # This feature validates PodSpecs from the validating webhook
     # against the K8s API Server.
     #
-    # When "allowed", the client may enable the behavior with the
-    # 'features.knative.dev/podspec-dryrun':'enabled' annotation.
+    # When "allowed", the server will not run the dry-run validation by default.
+    #   However clients may enable the behavior on an individual Service by
+    #   attaching the following metadata annotation: "features.knative.dev/podspec-dryrun":"enabled".
     # When "enabled"  the server will always run the extra validation.
```

This confused me at first, and I think it'd actually have been less confusing if we just had the "enabled" explanation first, e.g.

```suggestion
    # When "enabled"  the server will always run the extra validation.
    # When "allowed", the server will not run the dry-run validation by default.
    #   However clients may enable the behavior on an individual Service by
    #   attaching the following metadata annotation: "features.knative.dev/podspec-dryrun":"enabled".
```